doc/tutorials/introduction/building_tegra_cuda/building_tegra_cuda.markdown

```diff
@@ -0,0 +1,620 @@
+
+|Platform|Suggested value|Notes|
+|--------|---------------|-----|
+|DRIVE PX 2|6| |
+|Jetson TK1|3|If the build fails due to a compiler-related error, try again with a smaller number of threads. Also consider rebooting the system if it has been running for a long time since the last reboot.|
+|Jetson TX1|4| |
+|Ubuntu Desktop|7|The actual value will vary with the number of cores you have and the amount of physical memory. Because of the resource requirements of compiling the CUDA code, it is not recommended to go above 7.|
+
+Based on the value you select, build (assuming you selected 6):
+
+    $ make -j6
+
+By default, CMake hides the details of the build steps. If you need to see more detail about each compilation unit, etc., you can enable verbose output:
+
+    $ make -j6 VERBOSE=1
+
+Testing OpenCV
+--------------
+
+Once the build completes successfully, you have the option of running the extensive set of tests that OpenCV provides. If you did not clone the `opencv_extra` repository and specify the path to `testdata` in the `cmake` invocation, then testing is not recommended.
+
+### Testing under Linux
+
+To run the basic tests under Linux, execute:
+
+    $ make test
+
+This executes `ctest` to carry out the tests, as specified in CTest syntax within the OpenCV repository. The `ctest` harness takes many different parameters (too many to list here, see the manual page for CTest to see the full set), and if you wish to pass any of them, you can do so by specifying them in a `make` command-line parameter called `ARGS`:
+
+    $ make test ARGS="--verbose --parallel 3"
+
+In this example, there are two (2) arguments passed to `ctest`: `--verbose` and `--parallel 3`. The first argument causes the output from `ctest` to be more detailed, and the second causes `ctest` to run as many as three (3) tests in parallel. As with choosing a thread count for building, base any choice for testing on the available number of processor cores, physical memory, etc. Some of the tests do attempt to allocate significant amounts of memory.
+
+### Known Issues with Tests
+
+At present, not all of the tests in the OpenCV test suite pass. There are tests that fail whether or not CUDA is compiled, and there are tests that are only specific to CUDA that also do not currently pass. 
+
+__Note:__ There are no tests that pass without CUDA but fail only when CUDA is included.
+
+As the full lists of failing tests vary based on platform, it is impractical to list them here.
+
+Installing OpenCV
+-----------------
+
+Installing OpenCV is very straightforward. For the Linux-based platforms, the command is:
+
+    $ make install
+
+Depending on the chosen installation location, you may need root privilege to install.
+
+Building OpenCV 2.4.X {#building_opencv_24X}
```

Comment:
Doxygen produces warning here, it is kind of a bug ([related discussion](http://stackoverflow.com/a/13953587)). To avoid it you should make top section headers one level higher (`--------` -> `========`).

Since anchors are visible in the whole documentation, it is better to prefix them with the tutorial id (`building_opencv_24X` -> `tutorial_building_tegra_cuda_24X`, don't forget to update corresponding `@ref` command).

To make navigation easier I suggest adding a table of contents:
- add `@tableofcontents` command to the beginning ([example](https://raw.githubusercontent.com/opencv/opencv/master/doc/tutorials/introduction/documenting_opencv/documentation_tutorial.markdown))
- add anchors to each header you want see in the TOC
  _example with 1-level headers only_:
  ![image](https://cloud.githubusercontent.com/assets/3304494/20962269/b8ce970e-bc79-11e6-84b1-34730f70aa5a.png)